pkg/cache/context.go

```diff
@@ -951,6 +951,12 @@ func (ctx *Context) updateApplicationTags(request *AddApplicationRequest, namesp
 		}
 	}
 
+	// add maxApps resource info as an app tag
+	maxApps := utils.GetNamespaceMaxAppsFromAnnotation(namespaceObj)
+	if maxApps != "" {
+		request.Metadata.Tags[siCommon.AppTagNamespaceResourceMaxApps] = maxApps
```

Comment:
The comment of this function should include "namespace.resourcemaxapps". Besides, the "namespace.resourceguaranteed" looks like missing in the previous change.
```
// update application tags in the AddApplicationRequest based on the namespace annotation
// adds the following tags to the request based on annotations (if exist):
//   - namespace.resourcequota
//   - namespace.parentqueue
//   - namespace.resourceguaranteed
//   - namespace.resourcemaxapps
func (ctx *Context) updateApplicationTags(request *AddApplicationRequest, namespace string) {
```

Reply:
Thanks for pointing this, addressed in latest PR.